guides/micronaut-oracle-cloud-streaming/chess-game/java/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,3 @@
+kafka:
```

Comment:
why are these being added?

Reply:
Probably added by mistake in the  different merges